analysis/__main__.py

```diff
@@ -0,0 +1,43 @@
+import glob
+import sys
+
+import pandas as pd
+
+from heuristic.classes import Problem, Solution
+from .statistics import STATISTICS
+
+
+def main():
+    """
+    Prints several statistics of each instance and descriptive
+    statistics of the instances overall to a file.
+    """
+    if len(sys.argv) < 2:
+        raise ValueError(f"{sys.argv[0]}: expected file location.")
+
+    cumulative_df = pd.DataFrame()
```

Comment:
`instances` - cumulative _what_. It's the statistics for all instances. Statistics is clear from context, so just `instances` suffices.